src/runner.js

```diff
@@ -749,16 +749,18 @@ function testRunnerFactory(providerName, pluginFactories = [], config = {}) {
         /**
          * Skip alias
          * @param {String|*} [scope] - the movement scope
+         * @param {Number|*} [ref] - the item ref
+         * @param {String|*} [direction] - next/previous/jump
          * @fires runner#move
          * @returns {runner} chains
          */
-        skip(scope) {
+        skip(scope, ref, direction) {
             if (_.isFunction(provider.skip)) {
-                return providerRun('skip', scope);
+                return providerRun('skip', scope, ref, direction);
```

Comment:
I'll find more natural to have `skip(scope, direction, ref)` since `ref` can be undefined in most of the case execpt when the direction is a jump

Reply:
no ?

Reply:
Agreed, changed.